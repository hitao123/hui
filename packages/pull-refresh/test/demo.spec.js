describe('pull-refresh component', () => {
  it('should show pull-refresh', () => {

  });
});
