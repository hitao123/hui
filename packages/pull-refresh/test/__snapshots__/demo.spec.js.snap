// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div class="demo-pull">
  <div class="h-pull-refresh">
    <div class="h-pull-refresh__track" style="transition:0ms;transform:translate3d(0, 0px, 0);">
      <div class="h-pull-refresh__head">
        <!---->
        <!---->
        <!---->
      </div>
    </div>
  </div>
</div>
`;
